Cargo.toml

```diff
@@ -113,7 +124,13 @@ std = [
     "zeroize?/std",
 ]
 ssz = ["std", "dep:ethereum_ssz"]
-alloc = ["proptest?/alloc", "rand?/alloc", "serde?/alloc", "valuable?/alloc", "zeroize?/alloc"]
+alloc = [
+    "proptest?/alloc",
+    "rand?/alloc",
+    "serde?/alloc",
+    "valuable?/alloc",
+    "zeroize?/alloc",
+]
```

Comment:
needs explicit features for `der = ["dep:der", "alloc"]` and `subtle = ["dep:subtle"]